pkg/apis/extensions/types.go

```diff
@@ -35,6 +35,15 @@ import (
 	"k8s.io/kubernetes/pkg/util/intstr"
 )
 
+// The following identify resource constants for Kubernetes object types
+const (
+	// ReplicaSets, number
+	ResourceReplicaSets api.ResourceName = "replicasets"
```

Comment:
I think `extensions/replicasets` works fine for me.  Right now, validation https://github.com/kubernetes/kubernetes/blob/master/pkg/api/validation/validation.go#L2533 treats present of `/` to imply a non-standard resource which is odd for standard resources...